torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -264,7 +287,7 @@ c10::optional<at::Tensor> runTorchBackendForOnnx(
     updated_val = inputTensorValues[0];
     std::vector<int64_t> shape(inputTensorValues[1].sizes()[0], 0);
     auto shape_a = inputTensorValues[1].accessor<int64_t, 1>();
-    for (size_t i = 0; i < inputTensorValues[1].sizes()[0]; ++i) {
+    for (int64_t i = 0; i < inputTensorValues[1].sizes()[0]; ++i) {
```

Comment:
Why change from `size_t` to `int64_t`, i should be non-negative, right?

Reply:
Yes, but the type of inputTensorValues[1].sizes()[0] is **int64_t**. if not change,  the CI will throw the warnings for comparing different types.

Reply:
Sure, have added it. 

> Does squeeze op constant folding need update?

Yes, updated now. Thanks.

Reply:
In the latest modification. I have added assert inputTensorValues[1].sizes()[0] >= 0 and convert it into size_t type. This part CI is also succeed now.